### ISSUE TITLE
the application should create a playlist for any m3u file

### ISSUE THREAD

**Issue by bil-elmoussaoui** [gnumdk/lollypop#144]
Hello, 
when the application scans the library it's tried to get the tags of ```.m3u``` files 
```
** (lollypop:6332): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-l9VznPiA5r: Connexion refusée
cannot import name Secret, introspection typelib not found
Last.fm authentication disabled
Player::restore_state(): track missing
Can't get infos for  /media/Data/Music/Ed Sheeran - X/00_ed_sheeran-x-2014.m3u
Can't get infos for  /media/Data/Music/Enrique Iglesias - Sex And Love/00-enrique_iglesias-sex_and_love-(mexican_deluxe_edition)-cd-2014-wws.m3u
Can't get infos for  /media/Data/Music/Imagine Dragons - Night Visions/00-imagine_dragons-night_visions-2012.m3u
Can't get infos for  /media/Data/Music/Imagine Dragons - Smoke And Mirrors/000-imagine_dragons-smoke_and_mirrors-(deluxe_edition)-2cd-2015.m3u
Can't get infos for  /media/Data/Music/Indila -Mini World/(00) Mini World - 1nDD.m3u
Can't get infos for  /media/Data/Music/ Lacrim - Corleone/00-lacrim-corleone-cd-fr-2014-fr3sh.m3u
Can't get infos for  /media/Data/Music/Lana Del Rey - Ultraviolence/00_lana_del_rey-ultraviolence-2014.m3u
Can't get infos for  /media/Data/Music/Maroon 5 - Songs About Jane/000-maroon_5-songs_about_jane_(10th_anniversary_edition)-2012-2cd.m3u
Can't get infos for  /media/Data/Music/Maroon 5 - X/00_maroon_5-x-2014.m3u
Can't get infos for  /media/Data/Music/Michael Jackson - Xscape/00_michael_jackson-xscape-2014.m3u
Can't get infos for  /media/Data/Music/Pitbull - Globalization/00-pitbull-globalization-cd-2014-fr3sh.m3u
Can't get infos for  /media/Data/Music/Sia - 1000 Forms of Fear/Album/00 Playlist.m3u
Can't get infos for  /media/Data/Music/ Wiz Khalifa - Blacc Hollywood/00-wiz_khalifa-blacc_hollywood_(deluxe_edition)-cd-2014-fr3sh.m3u
Can't get infos for  /media/Data/Music/Inna - Summer Days/00. Inna - Summer Days (Deluxe Edition) (2014).m3u
Can't get infos for  /media/Data/Music/Nicki Minaj - The Pinkprint/00-nicki_minaj-the_pinkprint_(deluxe_edition)-cd-2014-fr3sh.m3u
Can't get infos for  /media/Data/Music/Nicki Minaj - Pink Friday/00-nicki_minaj-pink_friday_roman_reloaded-2012.m3u

````

**Comment by gnumdk** [comment]
:+1:

**Comment by gnumdk** [comment]
https://bugzilla.gnome.org/show_bug.cgi?id=750561

Need this bug to be fixed.

**Comment by gnumdk** [comment]
Fixed upstream but will have to wait for GTK 3.20 on Ubuntu LTS :-/

**Comment by geordieF** [comment]
Ubuntu (which has a much bigger share than debian) has 3.18.9 on stable. It would probably be a better idea to put out a newer version with the fix and just run a check when APT is run.

**Comment by gnumdk** [comment]
@geordieF Debian Stable is 3.14...

**Comment by gatlinnewhouse** [comment]
@geordieF Debian Stretch (the next stable version) comes out on June 17th of this year, or at least that's the goal right now.

**Comment by gnumdk** [comment]
Yes, I know ;) Will allow me to clean many part of Lollypop code!
